scripts/simple-build-ofi.sh

```diff
@@ -13,10 +13,10 @@
 set -e
 
 if [ -z "$SOS_VERSION" ] ; then
-    SOS_VERSION="v1.5.0"
+    SOS_VERSION="v1.5.1"
```

Comment:
Not sure, but should this be 1.5.2?

Reply:
Looking back at our last few releases, I think we keep this 1 minor release number behind the latest.  But the Release Checklist in the wiki could be more clear about this...  Are you ok with that protocol?  I think it's slightly easier to do it this way, then we don't have to worry about updating _after_ we drop the release.